scene/gui/graph_edit.cpp

```diff
@@ -2188,6 +2190,8 @@ void GraphEdit::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("is_right_disconnects_enabled"), &GraphEdit::is_right_disconnects_enabled);
 
 	ClassDB::bind_method(D_METHOD("_update_scroll_offset"), &GraphEdit::_update_scroll_offset);
+	ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::BOOL, "_is_in_input_hotzone", PropertyInfo(Variant::NODE_PATH, "graph_node"), PropertyInfo(Variant::INT, "slot_index"), PropertyInfo(Variant::VECTOR2, "mouse_position")));
```

Comment:
Please pass PropertyInfo(&GraphNode, "graph_node") directly?

Reply:
Okay. I've updated `PropertyInfo(Variant::NODE_PATH, "graph_node")` accordingly. I've also updated the virtual method call accordingly.

Reply:
I've got 11 failed checks, many point to GraphNode in `PropertyInfo(&GraphNode, "graph_node")` being an illegal use of this type as an expression.

Could this be resolved through `graph_node.h`?
```
./scene/gui/graph_node.h:37:7: note: declared here
class GraphNode : public Container {
      ^
scene/gui/graph_edit.cpp:2220:115: error: 'GraphNode' does not refer to a value
        ClassDB::add_virtual_method(get_class_static(), MethodInfo(Variant::BOOL, "_is_in_output_hotzone", PropertyInfo(&GraphNode, "graph_node"), PropertyInfo(Variant::INT, "slot_index"), PropertyInfo(Variant::VECTOR2, "mouse_position")));
```

Reply:
Maybe try `Variant::OBJECT`

Reply:
I've replaced `PropertyInfo(&GraphNode, "graph_node")` with `PropertyInfo(Variant::OBJECT, "graph_node")`.